skiko/src/commonMain/kotlin/org/jetbrains/skia/Image.kt

```diff
@@ -138,6 +138,36 @@ class Image internal constructor(ptr: NativePointer) : RefCnt(ptr), IHasImageInf
             return Image(ptr)
         }
 
+        /** Creates GPU-backed [org.jetbrains.skia.Image] from backendTexture associated with context.
```

Comment:
- Please use KDoc format of the comment:
```
/**
 *
 *
 */
```
- describe the parameters as in the original documentation in Skia